### PR TITLE
cleanup aliased elastic pillows

### PR DESCRIPTION
**Pull request by czue**
codependent with https://github.com/dimagi/pillowtop/pull/128

probably easiest read commit by commit. this has no real functional changes and mostly just removes a bunch of legacy/duplicated code

@snopoke @dannyroberts cc @emord

### CODE REVIEW

**Comment by czue** [comment]
(build fail doesn't look related)

**Comment by emord** [comment]
@czue i'll revert this https://github.com/dimagi/django-digest/pull/14

**Comment by czue** [comment]
thanks, kicked build

**Comment by czue** [comment]
:white_check_mark:

File: corehq/pillows/user.py [hunk 1]
```diff
@@ -52,26 +52,6 @@ def calc_meta(self):
     def get_unique_id(self):
         return USER_INDEX
 
-    def get_mapping_from_type(self, doc_dict):
```
Comment (benrudolph) [inline corehq/pillows/user.py]: this is great. i love that every word of the comment was copied too

**Comment by esoergel** [comment]
Wow :+1:

**Pull request merged**